examples/register_configs.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import hydra
+from typing import Any
+from omegaconf import OmegaConf
+
+import hydra_configs.torch.optim
+
+# registers all 'base' optimizer configs with configstore instance
+hydra_configs.torch.optim.register_configs()
```

Comment:
I think we can also add:
hydra_configs.torch.register_configs()
which will call register_configs for all the included modules.
(You can also add it later, when you add another register_configs function.

Reply:
Also:
since this is already in hydra_configs, I think we can just call the function register().

Reply:
all sounds good :+1: